core/src/UserSettingsDialog.html

```diff
@@ -303,6 +308,15 @@
       dispatch('close');
     }
   }
+
+  export function handleKeyUp(event, index) {
+    if (
+      event.keyCode === (KEYCODE_ENTER || KEYCODE_SPACE) &&
```

Comment:
I think that `event.keyCode === (KEYCODE_ENTER || KEYCODE_SPACE)` will only be true for ENTER and not space. Maybe you meant sth like this instead.  
```suggestion
      (event.keyCode === KEYCODE_ENTER || event.keyCode === KEYCODE_SPACE) &&
```